airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/source.py

```diff
@@ -156,11 +160,25 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
             # In order to support direct datetime string comparison (which is performed in incremental acceptance tests)
             # convert any date format to python iso format string for date based cursors
             self.update_field(record, self.cursor_field, lambda date: isoparse(date).isoformat())
+            # unnest cursor_field to handle normalization correctly
+            if self.unnest_cursor:
+                self.unnest_field(record, self.nested_object, self.cursor_field)
+            # unnest primary_key to handle normalization correctly
+            if self.unnest_pk:
+                self.unnest_field(record, self.nested_object, self.primary_key)
             yield record
 
         # sleep for 1-2 secs to not reach rate limit: 50 requests per minute
         time.sleep(60 / self.requests_per_minute)
 
+    @staticmethod
+    def unnest_field(record: Mapping[str, Any], unnest_from: Dict, cursor_field: str):
+        """
+        Unnest cursor_field to the root level of the record.
+        """
+        if unnest_from in record.keys():
```

Comment:
if unnest_from in record.keys():

--> 


if unnest_from in record:

Reply:
Corrected.